examples/helix-basic-tds/.env

```diff
@@ -16,19 +16,23 @@ SITE_HOST=www.basic-company-tds.localhost
 
 SITECORE_DOCKER_REGISTRY=scr.sitecore.com/sxp/
 SITECORE_TOOLS_REGISTRY=scr.sitecore.com/tools/
-SITECORE_VERSION=10.0-ltsc2019
-TOOLS_VERSION=10.0.0-1809
+SITECORE_VERSION=10.1-ltsc2019
+TOOLS_VERSION=10.1.0-1809
 
 SITECORE_ADMIN_PASSWORD=
 SQL_SA_PASSWORD=
 TELERIK_ENCRYPTION_KEY=
 SITECORE_IDSECRET=
 SITECORE_ID_CERTIFICATE=
 SITECORE_ID_CERTIFICATE_PASSWORD=
-SOLR_CORE_PREFIX_NAME=sitecore
+SOLR_CORE_PREFIX_NAME=basictds
```

Comment:
Can we keep this consistent across the examples and just remain `sitecore`? No reason for them to be unique in containers.